packages/adapters/bitcoin/src/connectors/LeatherConnector.ts

```diff
@@ -27,6 +27,10 @@ export class LeatherConnector extends SatsConnectConnector {
     })
   }
 
+  public override get imageUrl() {
+    return this.wallet?.icon?.replace('data:image/svg', 'data:image/svg+xml')
```

Comment:
Interesting, where do we set that value first time? 🤔  Is it only up to us the value of this icon or it's coming form third party package?

Reply:
It comes from leather extension itself 🫠